### PR TITLE
Remove old "get.docker.com" install script

### PR DESCRIPTION
**Pull request by thaJeztah**
This script is part of the Docker product, and no longer maintained in this repository.

The script has been updated to install Docker CE packages from https://download.docker.com, and is now maintained in the https://github.com/docker/docker-install repository.

ping @mlaventure @tianon @andrewhsu @tiborvass PTAL

### CODE REVIEW

**Review by tianon — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
oh! commit message is a bit funky, let me fix that :joy:

File: hack/release.sh [hunk 1]
```diff
@@ -295,18 +295,10 @@ EOF
 	fi
```
Comment (tiborvass) [inline hack/release.sh]: @thaJeztah i think you can remove the entire file.

File: hack/README.md [hunk 2]
```diff
@@ -18,14 +18,6 @@ Generates AUTHORS; a file with all the names and corresponding emails of
...
 this repository.
 
-## Install (install.sh)
-
-Executable install script for installing Docker. If updates to this are
-desired, please use hack/release.sh during a normal release. The following
-one-liner may be used for script hotfixes:
-
-- `aws s3 cp --acl public-read hack/install.sh s3://get.docker.com/index`
-
```
Comment (tiborvass) [inline hack/README.md]: you can also remove `## Release (release.sh)` and ` ## Vendor (vendor.sh)`

File: hack/release.sh (hunk 1 above)
Comment (thaJeztah) [inline hack/release.sh]: @tiborvass yes, I'm working on a separate branch to remove all rpm/deb/release related stuff. There's still some issues around that, so decided to keep this PR just for the `install.sh`

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**